constants/constants.tsx

```diff
@@ -85,6 +84,20 @@ export const ABOUT_2 = {
   dark: true
 };
 
+export const TRACK = {
+  heading: "get on track",
+  text: ["August 9-11","Our six custom learning tracks, workshops and challenges help you truly understand major topics in tech, and prepare you to apply them to solve complex, interdisciplinary problems in the future.", "Each learning track has three, hour-long workshops, and one post-workshop do-it-yourself coding challenge."],
+  tracks: [
+    {id: 0, title: "Python", icon: "img/PythonIcon.svg", info: "yes we have some good info here mmhmm"},
```

Comment:
yes we have some [good info here](https://docs.google.com/document/d/1S1W0nse79HzekyUZJOAXUwJuBVSMPoqjbTCMaEkaBBs/edit?usp=sharing) mmhmm